test/auth/auth_test.go

```diff
@@ -277,3 +277,60 @@ func getAccessToken(t *testing.T, weaviateUrl, user, pw string) (string, string)
 	require.Nil(t, err)
 	return tokens.AccessToken, tokens.RefreshToken
 }
+
+func TestApiKey(t *testing.T) {
```

Comment:
Here also test names are prefixed with `TestAuth_` so that one could also run only all tests from this file